static/index.html

```diff
@@ -88,14 +92,63 @@ <h1>nobody.live</h1>
           return a.outerHTML;
         }).join(', ');
       }
+      // execute find trainId every 60 seconds
+      var intervalId = window.setInterval(fetchTrainId, 60000)
+      var foundTrainId = false
+      let changingIn = 60
+      function fetchTrainId() {
```

Comment:
I'm not sure I understand why we're serving an ID, then fetching the stream data for that ID rather than just having a single `getRaid` endpoint that will always return the current stream to raid?

Reply:
Hmm. Maybe I'm not understanding. Are you saying that we should fetch and store the entire stream object in the js script instead of first getting the id and then finding the stream with that specific id? Then we can just load in that stream using `displayStream`. That seems much more reasonable than two GETs.

Reply:
Not fetch and store; upon click, make the fetch to some `/raid` endpoint that will return the current raid; no need to muck around with fetching IDs. No need to store anything, keep track of intervals client side, or anything -- just request when needed.

Reply:
How could the back-end know how to serve the same stream to multiple clients? Maybe if there was some interval on the app api that stored the stream whenever the first access of `/raid` is made for that interval. But you seem to imply that no storage is needed. Sorry that I am not quite getting it

And the interval would be kept track of on `app.py` instead of the client side? I'm a little confused about that.

Reply:
Yeah I mean an interval server side; you can just keep the last raid stream in memory and return it if it's fresher than N minutes old, or set a new one to return.